src/actions/receive-profile.js

```diff
@@ -655,12 +653,34 @@ export function doneSymbolicating(): Action {
   return { type: 'DONE_SYMBOLICATING' };
 }
 
-export function coalescedFunctionsUpdate(
-  functionsUpdatePerThread: FunctionsUpdatePerThread
-): Action {
-  return {
-    type: 'COALESCED_FUNCTIONS_UPDATE',
-    functionsUpdatePerThread,
+export function coalescedSymbolicationStep(
```

Comment:
Please add a bit of of documentation to this function, each action creator should be treated as a public API. Also, it would be nice to define "coalesced" in this context, as it's a bit non-obvious. I think it's being used in a more "computer science" way that doesn't immediately show up in search engine results.